src/packer.pkr.hcl

```diff
@@ -103,7 +103,12 @@ source "amazon-ebs" "example" {
 }
 
 build {
-  sources = ["source.amazon-ebs.example"]
+  sources = ["source.amazon-ebs.ubuntu_server"]
+
+  # We are adding this to avoid the failure to install aptitude during build time 
```

Comment:
Yes, that is what I'm referring to.

Reply:
Reference:
https://github.com/cisagov/ubuntu-server-packer/issues/12

Reply:
Thanks- I made some edits to #12.  Can you please update your [comment here](https://github.com/cisagov/ubuntu-server-packer/blob/99c815cdc134bcd03a79fae3b8437eae0713546f/src/packer.pkr.hcl#L108) so that it mentions #12?

Reply:
See commit https://github.com/cisagov/ubuntu-server-packer/pull/8/commits/6326e8b3d9bd4583f5cbd40dc4a39edc9fa9ee6c